src/main/kotlin/net/maizegenetics/phgv2/cli/CreateRefVcf.kt

```diff
@@ -191,15 +198,47 @@ class CreateRefVcf : CliktCommand(help = "Create a haplotype VCF for the referen
             // Load to an hvcf file, write files to user specified outputDir
 
             val hvcfFileName = "${refName}.h.vcf"
-            var localRefHVCFFile = outputDir + "/" + hvcfFileName
+            val tiledbHvcfDir = "${dbPath}/hvcf_files"
+
+            // Make a folder for the hvcf files
+            // ALl hvcf files loaded to this tiledb dataset will be saved in this subfolder
+            Files.createDirectories(Paths.get(tiledbHvcfDir)) // skips creating folders that already exist
+            var localRefHVCFFile = tiledbHvcfDir + "/" + hvcfFileName
 
             //  This is in VariantUtils - it exports the gvcf file.
             // Include the VCF ALT Header lines created in the loop above
             exportVariantContext(refName, fullRefVCList, localRefHVCFFile, myRefSequence!!,altHeaderLines)
             //bgzip and csi index the file
             val bgzippedGVCFFileName = bgzipAndIndexGVCFfile(localRefHVCFFile)
-            myLogger.info("${bgzippedGVCFFileName} created and stored to ${outputDir}")
+            myLogger.info("${bgzippedGVCFFileName} created and stored to ${tiledbHvcfDir}")
+
+            // Create the tiledb datasets if they don't exist
+            File(dbPath).mkdirs()
+            Initdb().createDataSets(dbPath)
+
+            // Load the Ref hvcf file to tiledb
+            val loadVcf = LoadVcf()
+            loadVcf.loadVcfFiles(tiledbHvcfDir,dbPath,"1") // default the threads to "1" for the ref.
```

Comment:
If you have the chance to have a valid default consider adding it to the loadVcfFiles function directly so you don't have the magic number showing up here.

Reply:
done